src/NuGet.Indexing/AzureDirectorySynchronizer.cs

```diff
@@ -104,7 +104,7 @@ private static void UnidirectionalSync(AzureDirectory sourceDirectory, Directory
 
             // we'll remove old files from both AzureDirectory's cache directory, as well as our destination directory
             // (only when older than 45 minutes - old files may still have active searches on them so we need a margin)
-            var referenceTimestamp = LuceneTimestampFromDateTime(DateTimeOffset.UtcNow.AddMinutes(-45));
+            var referenceTimestamp = LuceneTimestampFromDateTime(DateTime.UtcNow.AddMinutes(-45));
```

Comment:
Don't know what kind of UT framework exists for this code, but can we cover this fix with UTs?